packages/graphql/src/translate/translate-aggregate.ts

```diff
@@ -28,7 +28,7 @@ import { compileCypher } from "../utils/compile-cypher";
 function translateAggregate({ node, context }: { node: Node; context: Context }): [Cypher.Clause, any] {
     const { fieldsByTypeName } = context.resolveTree;
     const varName = "this";
-    let cypherParams: { [k: string]: any } = context.cypherParams ? { cypherParams: context.cypherParams } : {};
+    let cypherParams: { [k: string]: any } = context.cypherParams ? { ...context.cypherParams } : {};
```

Comment:
Maybe `Record<string, any>` or `Record<string, unknown>` is a better type for this?